### DISCUSSION TITLE
the audio cuts out azuracast

### DISCUSSION THREAD

**Discussion by rfolkperu** [AzuraCast/AzuraCast#6897] [General]
Hello, how are you? I have the following problem during the day, the autodj's music was working normally. After several hours, the autodj's music started to cut off. Why is this please help?

**Comment by BusterNeece** [comment]
We continue to get reports of this issue and are following it with Liquidsoap. If you have any logs or other information on how to reproduce the issue, please include them on our issue #6736.
